tests/conftest.py

```diff
@@ -258,6 +258,17 @@ def waitpkg_bin(test_fc_session_root_path):
     yield waitpkg_bin_path
 
 
+@pytest.fixture(scope="session")
+def msr_reader_bin(test_fc_session_root_path):
+    """Build a binary that reads msrs"""
+    msr_reader_bin_path = os.path.join(test_fc_session_root_path, "msr_reader")
+    build_tools.gcc_compile(
```

Comment:
hmm I think this defaults to `-static` in case host and guest libc differ. But if you wrote it in Python then you wouldn't have to worry about that or even compiling it.